src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,14 @@
-package core.basesyntax;
-
-public class MainApp {
-
-}
+package core.basesyntax;
+
+public class MainApp {
+    public static void main(String[] args) {
+        Truck truck = new Truck();
+        Excavator excavator = new Excavator();
+        Bulldozer bulldozer = new Bulldozer();
+        Machine[] machines = {truck, bulldozer, excavator};

```

Comment:
<img width="1065" alt="Screen Shot 2022-09-19 at 6 35 14 PM" src="https://user-images.githubusercontent.com/90692308/191056312-dd3eae76-f630-41a1-9974-1a4fe878edba.png">
Read the checklist more carefully

Reply:
fixed